query/src/main/scala/hmda/query/repository/institutions/InstitutionCassandraRepository.scala

```diff
@@ -0,0 +1,141 @@
+      s"parent_name," +
+      s"parent_city," +
+      s"parent_state," +
+      s"assets," +
+      s"other_lender_codes," +
+      s"top_holder_id_rssd," +
+      s"top_holder_name," +
+      s"top_holder_city," +
+      s"top_holder_state," +
+      s"top_holder_country) " +
+      s" VALUES " +
+      s"(?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)")
+  }
+
+  val statementBinder = (institution: InstitutionQuery, statement: PreparedStatement) =>
+    statement.bind(
+      institution.id,
+      new Integer(institution.agency),
+      new Integer(institution.filingPeriod),
+      new Integer(institution.activityYear),
+      institution.respondentId,
+      institution.institutionType,
+      new java.lang.Boolean(institution.cra),
+      institution.emailDomain1,
+      institution.emailDomain2,
+      institution.emailDomain3,
+      institution.respondentName,
+      institution.respondentState,
+      institution.respondentCity,
+      institution.respondentFipsStateNumber,
+      new java.lang.Boolean(institution.hmdaFilerFlag),
+      institution.parentRespondentId,
+      new Integer(institution.parentIdRssd),
+      institution.parentName,
+      institution.parentCity,
+      institution.parentState,
+      new Integer(institution.assets),
+      new Integer(institution.otherLenderCode),
+      new Integer(institution.topHolderIdRssd),
+      institution.topHolderName,
+      institution.topHolderCity,
+      institution.topHolderState,
+      institution.topHolderCountry
+    )
+
+  override def createTable(): ResultSet = {
+    val query =
+      s"""
+         |CREATE TABLE IF NOT EXISTS $keyspace.institutions(
+         |    id varchar PRIMARY KEY,
+         |    agency int,
+         |    period int,
+         |    activity_year int,
+         |    respondent_id varchar,
+         |    type varchar,
+         |    cra boolean,
+         |    email_1 varchar,
+         |    email_2 varchar,
+         |    email_3 varchar,
+         |    respondent_name varchar,
+         |    respondent_state varchar,
+         |    respondent_city varchar,
+         |    respondent_fips varchar,
+         |    hmda_filer boolean,
+         |    parent_respondent_id varchar,
+         |    parent_id_rssd int,
+         |    parent_name varchar,
+         |    parent_city varchar,
+         |    parent_state varchar,
+         |    assets int,
+         |    other_lender_codes int,
+         |    top_holder_id_rssd int,
+         |    top_holder_name varchar,
+         |    top_holder_city varchar,
+         |    top_holder_state varchar,
+         |    top_holder_country varchar
+         |);
+      """.stripMargin
+
+    session.execute(query)
+
+  }
+
+  override def dropTable(): ResultSet = {
+    val query = s"""
+      |DROP TABLE IF EXISTS $keyspace.institutions;
+    """.stripMargin
+
+    session.execute(query)
+  }
+
+  override def insertData(source: Source[InstitutionQuery, NotUsed]): Future[Done] = {
```

Comment:
Not right now. Both of those are convenience methods. Reading will happen when we introduce publication (down the road) and the write method is useful if we need to populate a panel file directly (i.e. for historical data)